usage/report.go

```diff
@@ -102,12 +102,14 @@ func (report *Report) processNode(node *html.Node) (string, *html.Node) {
 	text := ""
 	current := node.NextSibling
 
+	r, _ := regexp.Compile("<[^>]*>")
+
 	for current != nil {
 		var buf bytes.Buffer
 		w := io.Writer(&buf)
 		html.Render(w, current)
 
-		notags := strip.StripTags(buf.String())
+		notags := r.ReplaceAllString(buf.String(), "")
```

Comment:
👍

Reply:
Can you check if the regexp process the the example commands that we have in some files:
```
step crypto jwe decrypt \< message.json
```

Reply:
👍 Did my due diligence and we're all safe with this regex.